pkg/remotewrite/config.go

```diff
@@ -194,49 +194,49 @@ func parseEnvs(env map[string]string) (Config, error) {
 	}
 
 	// envconfig is not processing some undefined vars (at least duration) so apply them manually
```

Comment:
not related to this PR, but I think this comment and all of the `envconfig:` struct tags are not needed, it seems like the extension is not using the `envconfig` at all

Reply:
The attempt was to refactor for polishing the code in the next days because I didn't plan this PR. Anyway, now we have this PR, so it should be fine to address it directly here.